build.fsx

```diff
@@ -123,12 +123,9 @@ Target.create "Build" (fun _ ->
         ] |> File.deleteAll
 
         for project in ["src/FsLex/fslex.fsproj"; "src/FsYacc/fsyacc.fsproj"] do
-            DotNet.publish (fun opts -> { 
-                opts with 
-                    Common = { opts.Common with CustomParams = Some "/v:n" }
-                    Configuration = DotNet.BuildConfiguration.Release
-                    Framework = Some framework 
-            }) project
+            CreateProcess.fromRawCommandLine "dotnet" $"publish {project} -c Release /v:n -f {framework}"
+            |> Proc.run
+            |> ignore
```

Comment:
Are you sure that you can just ignore result of execution? What if process fail (build, test, etc)? I believe current code just ignore exit code and build will pass.

I think we should check exit code and fail if exit code is not zero, like this https://github.com/fsprojects/SwaggerProvider/blob/master/build.fsx#L78-L84